content/en/serverless/installation/nodejs.md

```diff
@@ -37,184 +37,139 @@ To install and configure the Datadog Serverless Plugin, follow these steps:
+  --capabilities CAPABILITY_AUTO_EXPAND CAPABILITY_IAM
 ```
-3. Run the build script:
-```
-yarn build    # Yarn users
-npm run build # NPM users
-```
-
-Deploy the macro to the same region as your Lambda functions. You need to deploy the macro in every region where you are monitoring Lambda functions:
-
-4. You will need an S3 bucket to store the CloudFormation artifacts for the macro. If you don't have one already, you can create one with:
-
-    ```bash
-      aws s3 mb s3://<bucket name>
-    ```
-
-5. Package the provided CloudFormation template (`macro_template.yml`). This includes a Lambda function and a CloudFormation macro resource. The provided template uses the AWS Serverless Application Model (SAM), so it must be transformed before deployment:
-
- ```bash
-    aws cloudformation package \
-        --template-file macro_template.yml \
-        --s3-bucket <your bucket name here> \
-        --output-template-file packaged.template
-    ```
-
-6. Deploy the packaged CloudFormation template to a CloudFormation stack:
-
-    ```bash
-    aws cloudformation deploy \
-        --stack-name datadog-cfn-macro \
-        --template-file packaged.template \
-        --capabilities CAPABILITY_IAM
-    ```
 
 The macro is now deployed and ready to use.
 
-### Install the Datadog Lambda Library
-
-Use the Datadog CloudFormation macro to ingest traces, logs, and enhanced metrics from your application without any code instrumentation. The macro automatically attaches the Datadog Lambda Library for Node.js and Python to your functions using layers. At deploy time, it generates new handler functions that wrap your existing functions and initializes the Lambda Library.
+### Instrument the Function
 
-To install the Datadog Lambda Library with the CloudFormation macro, follow these steps:
+In your `template.yml`, add the following under the `Transform` section, **after** the `AWS::Serverless` transform for SAM.
 
-1. In your `template.yml`, add the following:
-  ```
-  Transform:
-	  - Name: DatadogCfnMacro
-      Parameters:
-        enableDDTracing: true
-			  flushMetricsToLogs: true
-        stackName: !Ref "AWS::StackName"
-			  forwarderArn: "arn:aws:lambda:<REGION>:<ACCOUNT-ID>:function:datadog-forwarder"
-  ```
-  Find your Datadog Forwarder ARN in the [AWS Console][2]. For more information on installing the Forwarder, see the [official documentation][3].
-
-2. Redeploy your serverless application.
+```
+Transform:
+  - AWS::Serverless-2016-10-31
+  - Name: DatadogServerless
+    Parameters:
+      stackName: !Ref "AWS::StackName"
+      nodeLayerVersion: "<LAYER_VERSION>"
+      forwarderArn: "<FORWARDER_ARN>"
+      service: "<SERVICE>"
+      env: "<ENV>"
+```
 
+Replace `<SERVICE>` and `<ENV>` with appropriate values, `<LAYER_VERSION>` with the desired version of Datadog Lambda layer (see the [latest releases][4]), and `<FORWARDER_ARN>` with Forwarder ARN (see the [Forwarder documentation][2]).
```

Comment:
`service` and `env` are optional parameters, but seem required if a customer is just looking at this page - should we add a note for this?